components/Caretaker.js

```diff
@@ -52,7 +52,7 @@ const Caretaker = ({history, navigation}) => {
 
       <BottomSheet isVisible={isVisible} containerStyle={{}}>
         <Button buttonStyle={styles.buttonStyle} title='x' titleStyle={styles.titleStyle} onPress={() => setIsVisible(false)}></Button>
-        <AddTimer addTimer={addTimer}/>
+        <AddTimer addTimer={addTimer} setIsVisible={setIsVisible}/>
```

Comment:
Does this make the form close automatically on submit?

Reply:
It should, yes. I think it's been updated since, but this would be what it is/was for.